src/utilities/diagnosticsEventUtil.js

```diff
@@ -0,0 +1,3 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Util is redundant in this file name. You're already in a utilities folder.
Also, consider naming just diagnostics.js instead of tying it only to an event.

Reply:
fixed